.github/workflows/release.yaml

```diff
@@ -0,0 +1,33 @@
+name: Release changesets
+
+# This workflow will be triggered on push to the main branch
+# on:
+#   push:
+#     branches:
+#       - main
```

Comment:
I think we should uncomment that (we only want to release once PR is merged into main).

Reply:
@marekkirejczyk yes, our end goal is to have changesets only on PRs merged into main. It's just temporary.
My reasoning it that testing github actions on merge is painful. 

Let's say something would not work as we expect. To reproduce and retry we would need to merge something into main to make sure everything is ok. 

My idea is to split that into 3 PRs that would be merged in short period of time: 
- changeset release job that can be fired manually from github (this PR)
- npm release that can be fired manually from github (next PR)
- when everything fine than I will just remove manual trigger and exchange it with release on merge to main (final last PR)